contributors/devel/sig-scalability/hollow-node_simplified_template.yaml

```diff
@@ -33,19 +33,18 @@ spec:
         - containerPort: 10250
         - containerPort: 10255
         env:
-        - name: CONTENT_TYPE
-          valueFrom:
-            configMapKeyRef:
-              name: node-configmap
-              key: content.type
         - name: NODE_NAME
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
         command:
-        - /bin/sh
-        - -c
-        - /kubemark --morph=kubelet --name=$(NODE_NAME) --kubeconfig=/kubeconfig/kubelet.kubeconfig $(CONTENT_TYPE) --alsologtostderr --v=2
```

Comment:
Hmm, it looks like $CONTENT_TYPE was set incorrectly here. IIUC, it should be set as `--kube-api-content-type=$(CONTENT_TYPE)`. It will work as long as the `$(CONTENT_TYPE)` is  not set. If it's set, the kubemark will fail to parse flags and fail. Good catch!

What is interesting is that we use the same manifests in our CI/CD tests - https://github.com/kubernetes/kubernetes/blob/7b3322f677734027aa685dc6d5e57bc5cea7ed57/test/kubemark/resources/hollow-node_template.yaml#L61
I believe it works only because  `CONTENT_TYPE` is empty. Could you fix it there as well? Thanks!

Reply:
@mm4tt No problem. I will do that. Thanks for the review.